adapters/interactiveOffers/params_test.go

```diff
@@ -0,0 +1,41 @@
+package interactiveOffers
+
+import (
+	"encoding/json"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"testing"
+)
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, validParam := range validParams {
+		if err := validator.Validate(openrtb_ext.BidderInteractiveOffers, json.RawMessage(validParam)); err != nil {
+			t.Errorf("Schema rejected interactiveOffers params: %s", validParam)
+		}
+	}
+}
+
+func TestInvalidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, invalidParam := range invalidParams {
+		if err := validator.Validate(openrtb_ext.BidderInteractiveOffers, json.RawMessage(invalidParam)); err == nil {
+			t.Errorf("Schema allowed unexpected params: %s", invalidParam)
+		}
+	}
+}
+
+var validParams = []string{
+	`{"pubid":35}`,
+}
+
+var invalidParams = []string{
+	`{"pubid":"35"}`,
```

Comment:
I like that you have a test case for the incorrect type. I also recommend tests where the required `pubid` is missing and also combinations involving `tmax`.